app/assets/javascripts/spotlight/blocks/solr_documents_block.js

```diff
@@ -9,7 +9,7 @@ SirTrevor.Blocks.SolrDocuments = (function(){
 
     icon_name: "items",
 
-    autocomplete_url: function() { return this.$instance().closest('form[data-autocomplete-exhibit-catalog-index-path]').data('autocomplete-exhibit-catalog-index-path').replace("%25QUERY", "%QUERY"); },
+    autocomplete_url: function() { return this.$instance().closest('form[data-autocomplete-exhibit-catalog-path]').data('autocomplete-exhibit-catalog-path').replace("%25QUERY", "%QUERY"); },
```

Comment:
Line is too long.<br>Identifier 'autocomplete_url' is not in camel case.<br>Mixed double and single quotes.